front/app/containers/Admin/projects/project/permissions/components/ModeratorListRow.test.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import ModeratorListRow from './ModeratorListRow';
+import { render, screen } from 'utils/testUtils/rtl';
+import { IUserData } from 'services/users';
+
+const projectId = 'projectId';
+const projectModeratorRole = {
+  type: 'project_moderator' as const,
+  project_id: projectId,
+};
+const moderator: IUserData = {
+  id: 'moderatorId',
+  type: 'user' as const,
+  attributes: {
+    first_name: 'Johnny',
+    last_name: 'Goode',
+    invite_status: 'accepted',
+    slug: 'test-citizenlab',
+    locale: 'en',
+    highest_role: 'project_moderator',
+    roles: [projectModeratorRole],
+    bio_multiloc: {},
+    registration_completed_at: '2018-11-26T15:40:54.355Z',
+    created_at: '2018-11-26T15:41:19.782Z',
+    updated_at: '2018-11-26T15:41:19.782Z',
+    email: 'johnny@citizenlab.co',
+    confirmation_required: false,
+    unread_notifications: 0,
+  },
+};
+
+describe('<ModeratorListRow />', () => {
+  it('disables the delete button if the user is not an admin', () => {
+    render(
+      <ModeratorListRow
+        isLastItem={false}
+        projectId={projectId}
+        moderator={moderator}
+      />
+    );
+
+    const deleteButton = screen.getByRole('button', { name: /delete/i });
+    expect(deleteButton).toBeDisabled();
+  });
+
+  it('shows an enabled delete button when user is an admin', () => {
```

Comment:
We probably also need to look at why this test is passing when the actual app shows the button disabled

Reply:
Thanks for finding this one! I was passing the moderator of the row instead to the signed-in user in the isAdmin check. It works now and the test also fails with the old code.